app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+import datetime
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
```

Comment:
It is a good practice to use type annotations for function parameters. In your case, `cafe` parameter is missing a type annotation. It should be `cafe: Cafe`.

Reply:
I don't quite understand what you mean, I don't see the mistake.